pkg/scheduler/framework/job_updater.go

```diff
@@ -13,6 +13,8 @@ import (
 	"volcano.sh/volcano/pkg/scheduler/api"
 )
 
+var EnqueueExist bool
```

Comment:
Kindly to ask about why hard code enqueue action to make the judgement?

Reply:
Introduce ActionMap to store which action is configured by customer.